Corona-Warn-App/src/androidTest/java/testhelpers/BaseUITest.kt

```diff
@@ -15,6 +15,9 @@ abstract class BaseUITest : BaseTestInstrumentation() {
         Manifest.permission.WRITE_EXTERNAL_STORAGE
     )
 
+    @get:Rule
+    val systemUIDemoModeRule = SystemUIDemoModeRule()
+
```

Comment:
Super class has the rule by default , so it is no needed to do that for subclasses